### DISCUSSION TITLE
Google Next ’24 attendance

### DISCUSSION THREAD

**Discussion by ZackMattor** [GoogleCloudPlatform/cloud-foundation-fabric#2066] [General]
Hey all,

Just a general question to see if anyone is going to be attending Google Next '24 in Las Vegas and would want to meet up to discuss topics around this project. My past few weeks have been provisioning a new GCP organization using this foundation framework, and would love to get together with like-minded folks to nerd out over best-practices and the like.

Looking forward to it,
Zack Mattor

**Comment by juliocc** [comment]
We are mostly based out of EMEA and do not currently have plans to attend Next. However, we are always up for a good chat, so if you would like to connect over VC, please let us know.
